app/mailers/memberships/leave_zusatzsektion_mailer.rb

```diff
@@ -23,8 +24,8 @@ def placeholder_person_name
     @person.to_s
   end
 
-  def placeholder_sektion_name
-    @sektion_name
+  def placeholder_group_name
+    @section.display_name
   end
```

Comment:
In two other mailers it is still called `placeholder_sektion_name`:
```
app/mailers/people/neuanmeldungen_mailer.rb
39:  def placeholder_sektion_name

app/mailers/memberships/terminate_sac_membership_mailer.rb
26:  def placeholder_sektion_name
```

Is this OK or should we change these as well?